prelude/toolchains/msvc/vswhere.py

```diff
@@ -202,6 +202,76 @@ def write_tool_json(out, tool):
     out.write(j)
 
 
+# for use with the ewdk to grab the environment strings
+def get_env(ewdkdir: str):
+    """
+    Inspiration taken from the following:
+    http://pythonwise.blogspot.fr/2010/04/sourcing-shell-script.html (Miki Tebeka)
+    http://stackoverflow.com/questions/3503719/#comment28061110_3505826 (ahal)
+    """
+
+    # We need to write the script that will make the important variables available
+    # This gets run from the temporary build directory, so this does not pollute the sources
+    vcVarsExtract = os.path.join(os.path.dirname(os.path.realpath(__file__)),
+                "VcVarsExtract.bat")
+    if not os.path.exists(vcVarsExtract):
+        with open(vcVarsExtract, "w") as f:
+            f.write("@echo off\r\ncall %%1 %%2 %%3 %%4 %%5 %%6 %%7 %%8 %%9 > NUL\r\nset")
+
+    batch_file = os.path.join(ewdkdir, "BuildEnv", "SetupBuildEnv.cmd")
+
+    # This does not work as a list of strings, unfortunately
+    cmd = vcVarsExtract + ' "' + batch_file + '" ' + "amd64"
+
+    output = subprocess.check_output(cmd)
+    output = output.decode("utf-8")
+    env = {}
+    for line in output.split('\r\n'):
+        if line and '=' in line:
+            first,second = line.split("=", 1)
+            env[first] = second
+
+    return env
+
+
+def find_with_ewdk():
+    env = get_env(os.environ.get("EWDKDIR"))
+
+    installation_path = Path(env.get("VSINSTALLDIR"))
+    vc_tools_version = env.get("VCToolsVersion")
+    tools_path = installation_path / "VC" / "Tools" / "MSVC" / vc_tools_version
+    bin_path = tools_path / "bin" / "HostX64" / "x64"
+    lib_path = tools_path / "lib" / "x64"
+    include_path = tools_path / "include"
+
+    PATH = [bin_path]
+    LIB = [lib_path]
+    INCLUDE = [include_path]
+
+    ucrt = Path(env.get("UCRTContentRoot"))
+    ucrt_version = env.get("Version_Number")
+
+    if ucrt and ucrt_version:
+        PATH.append(ucrt / "bin" / ucrt_version / "x64")
+        LIB.append(ucrt / "lib" / ucrt_version / "ucrt" / "x64")
+        INCLUDE.append(ucrt / "include" / ucrt_version / "ucrt")
```

Comment:
`ucrt` is always set at this place.
```suggestion
    ucrt = Path(env["UCRTContentRoot"])
    ucrt_version = env.get("Version_Number")

    if ucrt_version:
        PATH.append(ucrt / "bin" / ucrt_version / "x64")
        LIB.append(ucrt / "lib" / ucrt_version / "ucrt" / "x64")
        INCLUDE.append(ucrt / "include" / ucrt_version / "ucrt")
```

Reply:
For this, I mostly copied the structure of the code that used `vswhere` to find components. I can change it to what you want.